diff --git a/node_modules/vuepress-theme-vdoing/components/BodyBgImg.vue b/node_modules/vuepress-theme-vdoing/components/BodyBgImg.vue
index 97b6b55..446a793 100644
--- a/node_modules/vuepress-theme-vdoing/components/BodyBgImg.vue
+++ b/node_modules/vuepress-theme-vdoing/components/BodyBgImg.vue
@@ -11,7 +11,7 @@ export default {
   data() {
     return {
       bgImg: '',
-      opacity: 0.5
+      opacity: 0.8
     }
   },
   mounted() {
diff --git a/node_modules/vuepress-theme-vdoing/components/Footer.vue b/node_modules/vuepress-theme-vdoing/components/Footer.vue
index 1b5e5c0..ba374d8 100644
--- a/node_modules/vuepress-theme-vdoing/components/Footer.vue
+++ b/node_modules/vuepress-theme-vdoing/components/Footer.vue
@@ -10,17 +10,8 @@
         target="_blank"
       ></a>
     </div>
-
-    <!--Vdoing主题遵循MIT协议，完全开源且免费。如果您对主题的修改并不大，希望您保留主题的链接。-->
-    Theme by
-    <a
-      href="https://github.com/xugaoyi/vuepress-theme-vdoing"
-      target="_blank"
-      title="本站主题"
-      >Vdoing</a
-    >
     <template v-if="footer">
-      | Copyright © {{ footer.createYear }}-{{ new Date().getFullYear() }}
+      Copyright © {{ footer.createYear }}-{{ new Date().getFullYear() }}
       <span v-html="footer.copyrightInfo"></span>
     </template>
   </div>
@@ -42,9 +33,9 @@ export default {
 <style lang='stylus'>
 // $mobileSidebarWidth = $sidebarWidth * 0.82
 .footer
-  padding 5rem 1.5rem 2.5rem
+  padding 2rem 1.5rem 2.5rem
   text-align center
-  color #666
+  color: #FFFFFF
   box-sizing border-box
   font-size 0.85rem
   transition all 0.2s ease
